docs/operations/metrics.md

```diff
@@ -64,6 +64,8 @@ Metrics may have additional dimensions beyond those listed above.
 |`sqlQuery/time`|Milliseconds taken to complete a SQL query.|`id`, `nativeQueryIds`, `dataSource`, `remoteAddress`, `success`|< 1s|
 |`sqlQuery/planningTimeMs`|Milliseconds taken to plan a SQL to native query.|`id`, `nativeQueryIds`, `dataSource`, `remoteAddress`, `success`| |
 |`sqlQuery/bytes`|Number of bytes returned in the SQL query response.|`id`, `nativeQueryIds`, `dataSource`, `remoteAddress`, `success`| |
+|`broker/serverview/init/time`|Time taken to initialize the broker server view.||Depends on the number of segments|
```

Comment:
maybe also add "you can use this metric to detect if your brokers are taking too long to start".

Reply:
Done. Thanks